packages/ui/react-ui/src/hooks/useCrossMintStatus.ts

```diff
@@ -26,7 +25,7 @@ interface IProps {
 
 const validateClientId = (clientId: string): boolean => {
     try {
-        const isValid = validate(clientId) || new PublicKey(clientId);
+        const isValid = validate(clientId);
```

Comment:
Since this is just a check for saving calls to the status endpoint, do you guys think we can live without the `PublicKey` from `@solana/web3.js`? @mPaella

Reply:
Well the thing is if we remove it, we'll fail to send valid IDs (in the form of public keys) to the server

Reply:
Actually, it should be fine: 

all the clients who used the old "public key" format for client IDs don't need to check the onboarding status. Only the new ones do. So I think you can remove this safely

Reply:
@mPaella will do that 👍